CHANGELOG.md

```diff
@@ -0,0 +1,65 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+ - `as_rust_extra_field` attribute for `AsRust` custom derive, can be used to specify initialize fields in the rust struct that are not present in the C one
+ - `c_repr_of_convert` attribute for `CReprOf` custom derive, can be used to perform custom conversion. Note that fields with this attribute are ignored by the `AsRust` custom derive.
+ - `target_name` attribute for `AsRust` and `CReprOf` custom derive, can be used if a field has different names in rust and C
+ 
+### Changed
+ - `RawPointerConverter` no longer has a blanket implementation for all types (this could to easily be misused leading to nasty errors), the trait as been reworked to include mut variants, and a custom derive has been added for it
+
+### Fixed
+ - Missing `Drop` implementation for CStringArray causing memory leaks
+
+### Removed
+ - Removed the `point_to_string` legacy function (this should have been marked `unsafe` and has no place in this crate)
+
+
+## [0.3.0] - 2017-10-05
```

Comment:
Did we do this in `2017` ?

Reply:
my bad... a copy paste problem from the keep a changelog file :P, should be fixed now